lib/api.js

```diff
@@ -32,6 +32,10 @@ api.declare({
   scopes:      [['secrets:set:<name>']],
   title:       'Set Secret',
   stability:    'stable',
+  cleanPayload: payload => {
```

Comment:
Isn't it safer to do: `cleanPayload: payload => _.defaults({secret: '(omitted)'}, payload),`